Explorer/Assets/DCL/PerformanceAndDiagnostics/Analytics/Systems/PerformanceAnalyticsSystem.cs

```diff
@@ -67,74 +76,72 @@ private void ReportPerformanceMetrics()
-
-                ["gpu_frame_time_percentile_5"] = gpuFrameTimeReport.Value.Percentiles[0] * NS_TO_MS,
-                ["gpu_frame_time_percentile_10"] = gpuFrameTimeReport.Value.Percentiles[1] * NS_TO_MS,
-                ["gpu_frame_time_percentile_20"] = gpuFrameTimeReport.Value.Percentiles[2] * NS_TO_MS,
-                ["gpu_frame_time_percentile_50"] = gpuFrameTimeReport.Value.Percentiles[3] * NS_TO_MS,
-                ["gpu_frame_time_percentile_75"] = gpuFrameTimeReport.Value.Percentiles[4] * NS_TO_MS,
-                ["gpu_frame_time_percentile_80"] = gpuFrameTimeReport.Value.Percentiles[5] * NS_TO_MS,
-                ["gpu_frame_time_percentile_90"] = gpuFrameTimeReport.Value.Percentiles[6] * NS_TO_MS,
-                ["gpu_frame_time_percentile_95"] = gpuFrameTimeReport.Value.Percentiles[7] * NS_TO_MS,
-            });
+                jsonObjectBuilder.Set("gpu_hiccups_in_thousand_frames", gpuFrameTimeReport.Value.Stats.HiccupCount);
+                jsonObjectBuilder.Set("gpu_hiccups_time", gpuFrameTimeReport.Value.HiccupsReport.HiccupsTime * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_hiccups_avg", gpuFrameTimeReport.Value.HiccupsReport.HiccupsAvg * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_hiccups_min", gpuFrameTimeReport.Value.HiccupsReport.HiccupsMin * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_hiccups_max", gpuFrameTimeReport.Value.HiccupsReport.HiccupsMax * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_min_frame_time", gpuFrameTimeReport.Value.Stats.MinFrameTime * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_max_frame_time", gpuFrameTimeReport.Value.Stats.MaxFrameTime * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_mean_frame_time", gpuFrameTimeReport.Value.Average * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_5", gpuFrameTimeReport.Value.Percentiles[0] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_10", gpuFrameTimeReport.Value.Percentiles[1] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_20", gpuFrameTimeReport.Value.Percentiles[2] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_50", gpuFrameTimeReport.Value.Percentiles[3] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_75", gpuFrameTimeReport.Value.Percentiles[4] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_80", gpuFrameTimeReport.Value.Percentiles[5] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_90", gpuFrameTimeReport.Value.Percentiles[6] * NS_TO_MS);
+                jsonObjectBuilder.Set("gpu_frame_time_percentile_95", gpuFrameTimeReport.Value.Percentiles[7] * NS_TO_MS);
+            }
+
+            var json = jsonObjectBuilder.Build();
+            analytics.Track(General.PERFORMANCE_REPORT, json);
+            jsonObjectBuilder.Release(json);
```

Comment:
we should be very careful with releasing reference-type objects to the pool, since they can be still in use (by the Segment analytics in this case). More probably, we can release it only after Flush. Currently, flush happens in 3 situations, where 1 is manual and 2 comes from `AnalyticsConfiguration.cs`: 
- flushSize == 20 (flushes when 20 messages are Tracked)
- flushInterval == 30 sec (flushes each 30 sec)
- flush on realm change (manual)